Python/import.c

```diff
@@ -99,6 +99,48 @@ _PyImportZip_Init(PyThreadState *tstate)
     return _PyStatus_ERR("initializing zipimport failed");
 }
 
+
+/* Debugging functions */
+
+// Print the message to stderr if -v/PYTHONVERBOSE is turned on
+// Only allow one args in message
+static void
+_verbose_message(const char *message, const char *args)
+{
+    PyThreadState *tstate = _PyThreadState_GET();
+    int verbose = _PyInterpreterState_GetConfig(tstate->interp)->verbose;
+
+    if (verbose) {
+        // confirm the prefix of message matches any keyword
+        bool startswith_keywords = false;
+
+        if (strlen(message) >= 1 && strncmp(message, "#", 1) == 0) {
+            startswith_keywords = true;
+        }
+
+        if (strlen(message) >= 7 && strncmp(message, "import ", 7) == 0) {
+            startswith_keywords = true;
+        }
+
+        if (startswith_keywords) {
```

Comment:
I'd still suggest changing this to a single `if` clause, to avoid the extra variable and get the benefits of short-circuiting (without the need to always do two `strlen()` and two `strncmp()` calls, as:

```c
if (strlen(message) >= 1 && (
    strncmp(message, "#", 1) == 0 || strncmp(message, "import ", 7) == 0
)) {
   ...
```

Reply:
I thought it would be clearer if I list all the cases out, but as you said using a single `if` can make it efficient.

Reply:
Another thing I learned from your code is that changing the organization of conditions also makes the code clearer!